src/app/components/harvest/screens/uploading/batch-uploading.component.html

```diff
@@ -28,6 +28,13 @@ <h3>Uploading Files</h3>
           >
           and we will check them for you
         </li>
+        <li>
+          We advise that you use this
+          <a href="https://github.com/QutEcoacoustics/workbench-client/wiki/Harvest-Proposal#WinSCP-Background-Transfer">
```

Comment:
```suggestion
          <a href="https://winscp.net/eng/docs/ui_pref_background#background_transfers">
```

Reply:
The amendment has been made, and the blob has been has been removed from the git history